src/Elewant/Webapp/Infrastructure/ProophProjections/HerdReadModel.php

```diff
@@ -150,33 +145,25 @@ public function onElePHPantWasAbandonedByHerd(ElePHPantId $elePHPantId): void
         );
     }
 
-    /**
-     * @param HerdId            $herdId
-     * @param Breed             $breed
-     * @param DateTimeImmutable $desiredOn
-     *
-     * @throws DBALException
-     */
     public function onBreedWasDesiredByHerd(HerdId $herdId, Breed $breed, DateTimeImmutable $desiredOn): void
     {
         try {
             $this->connection->insert(
                 self::TABLE_DESIRED_BREEDS,
                 [
-                    'herd_id'    => $herdId->toString(),
-                    'breed'      => $breed->toString(),
+                    'herd_id' => $herdId->toString(),
+                    'breed' => $breed->toString(),
                     'desired_on' => $desiredOn->format('Y-m-d H:i:s'),
                 ]
             );
-        } catch (UniqueConstraintViolationException $e) {
+        } catch (DBALException $e) {
```

Comment:
Hmm... I've seem to mistakenly concluded that `UniqueConstraintViolationException` isn't thrown anymore, but (since Doctrine ORM 2.6) it's hidden in some wrapping technique. I'll fix this so the original behavior is maintained.